src/key_value_store/server/kvstore_server.cpp

```diff
@@ -31,10 +46,82 @@ KeyValueStoreImpl::get(ServerContext *context,
 Status KeyValueStoreImpl::remove(ServerContext *context,
                                  const RemoveRequest *request,
                                  RemoveReply *response) {
+  Status status;
   bool removeSuccessBool = store_.Remove(request->key());
   if (removeSuccessBool) {
-    return Status::OK;
```

Comment:
In general, I prefer to use `return` instead of setting a variable just so when you are walking through the code later it is easier to follow and understand how the program is exiting.  That is totally personal preference though and I don't think there is anything wrong with this this code.

Reply:
Oh hahaha I realized that I was setting variables and returning once in all my other functions so changed this for consistency. I generally don't mind either way so just went with what I had already done.

Reply:
yeah makes sense, sounds good